src/main/kotlin/lotto/util/LottoGenerator.kt

```diff
@@ -1,7 +1,18 @@
 package lotto.util
 
 import lotto.domain.Lotto
+import lotto.domain.Lottos
 
 interface LottoGenerator {
+
+    fun getLottos(count: Int): Lottos {
+        val lottoList = buildList {
+            repeat(count) {
+                add(getLotto())
+            }
+        }
```

Comment:
다음처럼 표현할 수도 있을 것 같아요.
```suggestion
        val lottoList = List(count) {
            getLotto()
        }
```

Reply:
오호 Array만 생성과 동시에 초기화가 되는줄 알았는데 List도 가능하군요! ㅋㅋㅋㅋ